pkgs/misc/vscode-extensions/default.nix

```diff
@@ -107,6 +107,18 @@ let
         };
       };
 
+      formulahendry.auto-rename-tag = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "auto-rename-tag";
+          publisher = "formulahendry";
+          version = "0.1.5";
```

Comment:
The latest release seems to be [0.1.6](https://github.com/formulahendry/vscode-auto-rename-tag/blob/master/CHANGELOG.md#016-2021-01-06).

Reply:
My bad came out 2 days ago and VSCode hasn't given me the update yet. Should be fine now.